api/policies/hasValidMoveBody.js

```diff
@@ -27,6 +27,20 @@ module.exports = function (req, res, next) {
       }
     }
       return next();
+
+    case MoveType.SCUTTLE: {
+      if (!cardId || !targetId) {
+        return res.badRequest({ message: 'Must specify a card' });
+      }
+
+      const invalidCardId = [cardId, targetId].some(id => !DeckIds.includes(id));
```

Comment:
Okay I now see that how you're using the invalidCardId in the error message only works with .find() not .some(). My mistake